eva-lib/src/main/java/uk/ac/ebi/eva/lib/repository/VariantEntityRepository.java

```diff
@@ -60,4 +63,11 @@ Long countByChromosomeAndStartAndReferenceAndAlternate(String chromosome, int st
 
     @Query(value = "{'chr': ?0, 'start': ?1, 'ref': ?2}", count = true)
     Long countByChromosomeAndStartAndReference(String chr, int start, String ref);
+
+    @Query(value = "{'chr': ?0, 'start': ?1, 'alt': ?2, 'files.sid': {$in : ?3}}}", count = true)
+    Long countByChromosomeAndStartAndEndAndAltAndStudyIn(String chr, int start, String alt, List<String> studyIds);
+
+    @Query(value = "{'chr': ?0, 'start': ?1, 'type': ?2, 'files.sid': {$in : ?3}}}", count = true)
```

Comment:
I wonder, can we use constants to define this string? That way the queries wouldn't need to be modified if the database schema changes.